user/imports.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Importing shared build configuration
+
+layout: en
+---
+
+The main source of configuration for your build is the `.travis.yml` file stored in your repository. Now you can import up to five configuration snippets into your `.travis.yml`, so you can update multiple repositories making only one change.
+
+> BETA Configuration imports is currently in beta. Please leave feedback on the [Community forum](https://travis-ci.community/c/early-releases).
+{: .beta }
+
+## Example
+
+Instead of specifying which versions of Ruby to test against in multiple files, you can define them in a snippet:
+
+```yaml
+rvm:
+  - 2.5
+  - 2.6
+```
+{: data-file="rubies.yml"}
+
+You can then `import` that snippet into your `.travis.yml`. The following configuration imports the file `rubies.yml` from the `main` branch of the `shared-configs` repository of the `travis-ci` account:
+
+```yaml
+import: travis-ci/shared-configs/rubies.yml@main
+script: bundle exec rake
+```
+{: data-file=".travis.yml"}
+
+Resulting in the following configuration:
+
+```yaml
+rvm:
+  - 2.5
+  - 2.6
+script: bundle exec rake
+```
+{: data-file=".travis.yml"}
+
+## Import key
+
+You can import up to 5 YAML sources/snippets into one `.travis.yml`, and you can of course reuse those snippets in multiple repositories.
+
+The format of the `import` source is `<account>/<repository>/<path>@<ref>` in whic `<ref>` can be any valid Git reference, such as a commit sha, branch name, or tag name. If you do not specify `<ref>`, Travis CI will fetch the latest version of the default branch in the repository.
+
+> Public repositories can import sources from public repositories, but not private repositories. Private repositories can import sources from both public and private repositories.
+
+Import a single snippet:
+
+```yaml
+#import a single file
+import: travis-ci/build-configs/rubies.yml@main
+```
+{: data-file=".travis.yml"}
+
+Import multiple snippets:
+
+```yaml
+# import multiple files
+import:
+  - travis-ci/build-configs/rubies.yml@adf1235
+  - travis-ci/build-configs/other.yml@v1
+```
+{: data-file=".travis.yml"}
+
+By default, if the same keys are present in multiple imported snippets, they are overwritten with the last configuration imported. But you can also specify [deep_merge](#merge-mode).
+
+## Merge mode
+
+The default merge `mode` is `merge` which performs a shallow merge.
+This means that root level sections (keys) defined in your `.travis.yml` will be overwritten by root level sections (keys) that are also present in the imported yaml file.
+Merge modes are specific to each source.
+
+You can also use `deep_merge` and `replace`:
```

Comment:
i would not mention the merge mode `replace` here. for imports there's not a single use case ... the importing config would always entirely replace the imported one, making the whole import pointless :)